app/utils/api.js

```diff
@@ -0,0 +1,11 @@
+
+export function getProducts(authHeader) {
```

Comment:
From the [Folder Structure](https://github.com/asimolmez/react-native-folder-structure) in the README this should be in app/helper/api